src/utils/chunk.ts

```diff
@@ -61,10 +61,52 @@ export function splitPaths(paths: string[]): string[] {
   return paths.map((m) => m.split(PathSplitCharacters)).flat();
 }
 
-export type FileFilter = { include?: string; exclude?: string; limit?: number; group?: number; groupSize?: string };
-export async function getFiles(paths: string[], args: FileFilter = {}): Promise<string[][]> {
+export type FileFilter = {
+  include?: string;
+  exclude?: string;
+
+  /**
+   * Limit the number of the output files
+   *
+   * @default -1 - No limit
+   */
+  limit?: number;
+
+  /**
+   * Group files into this number of items group
+   *
+   * @default -1 - No limit
+   */
+  group?: number;
+
+  /**
+   * Group the files into this size groups, see {@link parseSize}
+   *
+   * @example
+   * ```
+   * 5GB // 5 GB chunks
+   * ```
+   *
+   * @default -1 - No limit
+   */
+  groupSize?: string;
+
+  /**
+   * Files less than size are ignored
+   *
+   * @example
+   *
+   * ```typescript
+   * if(file.size < sizeMin) continue
+   * ```
+   * @default 1
+   */
+  sizeMin?: number;
+};
+export async function getFiles(paths: string[], args: FileFilter = { sizeMin: 1 }): Promise<string[][]> {
```

Comment:
Why not setting all the default value here?

Reply:
This code block would be only triggered if args is undefined. I need to remove `sizeMin` from it, it was only put there for some temporary testing.